### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by AnnaLoshak**
Please check my code

### CODE REVIEW

File: src/test/java/Group4/Anna.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+           if(word1.equals(words[i]))
+               for (int j = 0; j < words.length; j++)
+              if(word2.equals(words[j]))
+                  if(Math.abs(j-i)<distance)
+                      distance=Math.abs(j-i);
+       return distance;
+
+    }
+}
+///**
```
Comment (BadamkhandB) [inline src/test/java/Group4/Anna.java:25]: Looks great

**Review by BadamkhandB — COMMENTED** [review]
Good

**Review by BadamkhandB — APPROVED** [review]

File: src/test/java/Group4/Anna.java (hunk 1 above)
Comment (KarinaFrei) [reply src/test/java/Group4/Anna.java:25]: Good job.

**Review by ZaierAouani — APPROVED** [review]
Great!

**Review by Aramat16ukr — APPROVED** [review]
great work

**Review by VeronikaKrasnokutska — APPROVED** [review]
Good!

**Review by KarinaFrei — APPROVED** [review]

**Review by natebell510 — APPROVED** [review]
A

**Review by BadamkhandB — APPROVED** [review]

**Review by msebbata — APPROVED** [review]
all good

**Pull request merged**